orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/persistence/ExecutionRepository.groovy

```diff
@@ -32,6 +32,8 @@ interface ExecutionRepository {
   void cancel(String id)
   void cancel(String id, String user)
 
+  void forceCancel(String id, String user)
```

Comment:
Do you think it would make more sense to move the check for live Orca node inside the `ExecutionRepository.cancel` method? I didn't because it was an interface but I see we only have one implementation.